purerl/0.0.15.nix

```diff
@@ -0,0 +1,19 @@
+{ pkgs }:
+
+import ./mkPurerlDerivation.nix ( rec{
+  inherit pkgs;
+
+  version = "0.0.15";
+
+  src =
+    if pkgs.stdenv.isDarwin then
+      pkgs.fetchurl {
+        url = "https://github.com/purerl/purerl/releases/download/v${version}}/macos.tar.gz";
```

Comment:
This doesn't look right

Reply:
Got burned by nix-prefetch : [
fixed now